build-tools/pnpm-lock.yaml

```diff
@@ -8096,92 +8163,91 @@ snapshots:
       - bluebird
       - supports-color
 
-  '@oclif/core@4.0.30':
+  '@oclif/core@4.8.0':
     dependencies:
       ansi-escapes: 4.3.2
-      ansis: 3.3.2
+      ansis: 3.17.0
       clean-stack: 3.0.1
       cli-spinners: 2.9.2
       debug: 4.4.3(supports-color@8.1.1)
       ejs: 3.1.10
       get-package-type: 0.1.0
-      globby: 11.1.0
       indent-string: 4.0.0
-      is-wsl: 3.1.0
-      lilconfig: 3.1.2
+      is-wsl: 2.2.0
```

Comment:
Ok, it does seem like oclif decided to push this dep back, and the is-docker "regression" stems from this too. I guess they're fine.